livestream/configs/configs.go

```diff
@@ -33,16 +33,18 @@ type Config struct {
 }
 
 type KafkaConfig struct {
-	Brokers string `mapstructure:"brokers"`
-	Topic   string `mapstructure:"topic"`
-	GroupID string `mapstructure:"group_id"`
+	Brokers               string `mapstructure:"brokers"`
+	Topic                 string `mapstructure:"topic"`
+	SessionRecordingTopic string `mapstructure:"session_recording_topic"`
+	GroupID               string `mapstructure:"group_id"`
 }
 
 func InitConfigs(filename, configPath string) {
 	viper.SetConfigName(filename)
 	viper.AddConfigPath(configPath)
 
 	viper.SetDefault("kafka.group_id", "livestream")
+	viper.SetDefault("kafka.session_recording_topic", "session_recording_snapshot_item_events")
```

Comment:
i'm assuming we're ok to default to this value :see_no_evil:  
  
the only thing that occurs to me now is i bet events and replay are on different brokers!